### DISCUSSION TITLE
License for some of the libAtom code files

### DISCUSSION THREAD

**Discussion by mcaroba** [libAtoms/QUIP#287] [Q&A]
I am in the process of opening up the TurboGAP repo. For interoperability with QUIP+GAP, I reused the splines code to be used with core potentials (src/libAtoms/Spline.f95). There is no authorship/license/copyright info in that file besides the generic statement at the very top. Other files state the author and license (e.g., GPL v3) but not this one. Can someone please let me know which copyright notice and license info I should use when redistributing that file with TurboGAP?

**Comment by gabor1** [comment]
I seem to recall that I typed that in from Numerical Recipes...

-- Gábor

Gábor Csányi
Professor of Molecular Modelling
Engineering Laboratory, University of Cambridge
Pembroke College Cambridge

Pembroke College supports CARA. A Lifeline to Academics at Risk.  http://www.cara.ngo/

**Reply by mcaroba** [reply]
Yes, it says so on the file, I didn't realize what it means until I googled it. It says "partially based on Numerical Recipes". What does this mean license-wise?

**Comment by gabor1** [comment]
hm… I think we may have to rewrite that bit of code!

http://numerical.recipes/licenses/


-- Gábor

Gábor Csányi
Professor of Molecular Modelling
Engineering Laboratory, University of Cambridge
Pembroke College Cambridge

Pembroke College supports CARA. A Lifeline to Academics at Risk.  http://www.cara.ngo/

**Reply by mcaroba** [reply]
Yes, I thought so. I can take a look, and come up with something that gives the same results. I suppose we can also use the same algorithm, unless they have patented it.

**Comment by gabor1** [comment]
Not patented Splines I don't think. 

-- Gábor

**Comment by noambernstein** [comment]
No, they’re just cubic splines
